### ISSUE TITLE
Search filter "in here" only applies together with search term

### ISSUE THREAD

**Issue by kulmann** [owncloud/web#9970]
## Describe the bug
The search filter "in here" from the search input field only applies together with a search term, not with empty search term but other filters.

## Steps to reproduce
1. Have two "test.md" files, one in your personal space and one in a project space
2. Navigate into the project space
3. Select "in here" in the search input field and trigger the search (enter / click on the magnifier icon)
4. Select a media type filter for "Documents" (which includes markdown)

## Expected behavior
Search result only shows documents from the project space

## Actual behavior
Search result also shows the "test.md" file from your personal space. Adding "test" as a search term successfully adds the "in here" location filter as well.

**Comment by jacob-nv** [comment]
Any ideas how to go about this?
Having empty query removes "scope", which is why empty search with "in here" searches everywhere.
Tried forcing the search with empty query, but that throws error from WebDAV

**Comment by lookacat** [comment]
@kulmann i think then we either can't fix it or we need a backend ticket

**Comment by kulmann** [comment]
Correct, I'd say this is a backend bug. The only thing to do in the frontend would be to remove this line:
https://github.com/owncloud/web/blob/22ea1b4484b28302a542ccc08403e2570c3e350b/packages/web-app-search/src/portals/SearchBar.vue#L223


But then this backend issue happens:
https://github.com/owncloud/ocis/issues/7812
